src/main/java/fr/foodlimit/api/user/UserService.java

```diff
@@ -1,28 +1,26 @@
 package fr.foodlimit.api.user;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-
 @Service
 public class UserService {
 
-  private final Map<String, User> db;
+  @Autowired
+  UserRepository userRepository;
 
   public UserService() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)